.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+src/test/*
```

Comment:
there will be a task to write tests for the fruit-shop :) just manually don't add changes if you already made some. You may have some problems later on to add files that were ignored